api.rst

```diff
@@ -1452,6 +1452,68 @@ You can use a tool like `Swagger UI <https://swagger.io/tools/swagger-ui/>`_ to
 
   The OpenAPI information can go out of date as the schema changes under a running server. To learn how to refresh the cache see :ref:`schema_reloading`.
 
+.. _options_requests:
+
+OPTIONS
+=======
+
+You can verify which HTTP methods are allowed on endpoints for tables and views using an OPTIONS request. These methods are allowed depending on what operations *can* be done on the table or view, not on the database permissions assigned to them.
+
+CORS
+----
+
+PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
+
+Tables
+------
```

Comment:
One issue with this heading, its URL fragment is simply "tables":
https://postgrest--407.org.readthedocs.build/en/407/api.html#tables

That might conflict with a future fragment. Maybe rename it to `options-tables`:
```suggestion

.. _options_tables:

Tables
------
```